ta4j-core/src/main/java/org/ta4j/core/indicators/helpers/DifferencePercentage.java

```diff
@@ -0,0 +1,78 @@
+/*
+  The MIT License (MIT)
+
+  Copyright (c) 2014-2017 Marc de Verdelhan, Ta4j Organization & respective authors (see AUTHORS)
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy of
+  this software and associated documentation files (the "Software"), to deal in
+  the Software without restriction, including without limitation the rights to
+  use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+  the Software, and to permit persons to whom the Software is furnished to do so,
+  subject to the following conditions:
+
+  The above copyright notice and this permission notice shall be included in all
+  copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+  FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+  COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+  IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+  CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+package org.ta4j.core.indicators.helpers;
+
+import org.ta4j.core.Indicator;
+import org.ta4j.core.indicators.CachedIndicator;
+import org.ta4j.core.num.Num;
+
+/**
+ * Difference Change Indicator.
+ * <p>
+ * Get the difference in percentage from the last time the threshold was reached.
+ * <p>
+ * Or if you don't pass the threshold you will always just get the difference from the precious value.
```

Comment:
Maybe the indicator should return the difference **in percentage** (like it does with threshold)? I think it is a little bit confusing if it returns absolute values on the one hand and relative values on the other hand

Reply:
It's actually doing exactly that (correct me if i'm misunderstanding you), i've updated that comment and the test to make it more clear. E.g. I've changed the test case to 1000 instead of 100, this makes it easier to see that from 1000 -> 1010 will result in 1(%) return value.